### DISCUSSION TITLE
I want to automatically send hl7 message to inform Java listening  after receive Archive

### DISCUSSION THREAD

**Discussion by lxb0425** [dcm4che/dcm4chee-arc-light#3980] [Q&A]
I want to automatically send hl7 message to inform Java listening  after receive Archive 

I use as follwow
https://github.com/dcm4che/dcm4chee-arc-light/wiki/Synchronize-external-HL7-Receivers-on-Patient-Information-Updates

But it did not work  can help me ？

**Comment by vrindanayak** [comment]
@lxb0425 : What exactly are your use cases and configurations?
